Libraries/Alert/Alert.js

```diff
@@ -122,6 +124,8 @@ class Alert {
             cancelButtonKey = String(index);
           } else if (btn.style === 'destructive') {
             destructiveButtonKey = String(index);
+          } else if (btn.isPreferred) {
```

Comment:
@robbie-c shouldn't this just be an if? It shouldn't depend on the style?

Reply:
That's fair, whilst preferred and destructive seem mutually exclusive visually (it's been a while since I wrote this so I don't remember which one "wins" if you apply both), if we're just passing through flags to the iOS API in an unopionated way it makes sense to match it, so I'll make this change.